src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulkscan/transformation/mappers/ChildrenInfoMapper.java

```diff
@@ -28,23 +32,19 @@
         public static final String COUNTRY = "CountryOfResidenceChild";
     }
 
-    private ChildrenInfoMapper() {
-        // don't
-    }
-
     public static void applyMappings(List<OcrDataField> ocrDataFields, Map<String, Object> modifiedMap) {
         ChildrenInfo children = new ChildrenInfo();
         for (int i = 1; isChildInfoPopulated(i, ocrDataFields); i++) {
             children.addChild(mapChild(i, ocrDataFields));
         }
 
-        if (children.size() > 0) {
+        if (!children.isEmpty()) {
             modifiedMap.put("childrenInfo", children);
         }
```

Comment:
Do we tend to do single line if statements such as

``` 
!children.isEmpty() ? modifiedMap.put("childrenInfo", children);
```

Reply:
done